airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessBuilderFactoryPOC.java

```diff
@@ -24,116 +24,31 @@
 
 package io.airbyte.workers.process;
 
-import com.google.common.collect.Lists;
 import io.airbyte.commons.io.IOs;
-import io.fabric8.kubernetes.api.model.Pod;
-import io.fabric8.kubernetes.api.model.PodList;
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.ConfigBuilder;
 import io.fabric8.kubernetes.client.DefaultKubernetesClient;
 import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.utils.HttpClientUtils;
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.io.PrintWriter;
-import java.nio.file.Path;
-import java.util.List;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
-import org.apache.commons.lang3.RandomStringUtils;
+import okhttp3.OkHttpClient;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class KubeProcessBuilderFactoryPOC {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(KubeProcessBuilderFactoryPOC.class);
-
-  private static final KubernetesClient KUBE_CLIENT = new DefaultKubernetesClient();
-
-  // todo: this should really be cached
-  public static String getCommandFromImage(String imageName) throws IOException {
```

Comment:
I moved all these functions to `KubePodProcess` as there was a hidden cyclic dependency. The process builder factory called the kube pod process class that called the factory back for these static functions when creating the process.